python/understack-workflows/understack_workflows/main/sync_server.py

```diff
@@ -77,3 +77,8 @@ def main():
     response = client.update_node(node.uuid, patches)
     logger.info(f"Patching: {patches}")
     logger.info(f"Updated: {response}")
+    return ironic_node.provision_state
+
+
+if __name__ == "__main__":
+    sys.stdout.write(main())
```

Comment:
This also won't work with how Python scripts are installed with a Python package. The `__main__` part isn't called and instead the function is called directly, which shouldn't return anything.